assets/js/modules/tagmanager/components/settings/SettingsForm.js

```diff
@@ -41,15 +66,36 @@ export default function SettingsForm() {
 			<FormInstructions />
 
 			<div className="googlesitekit-setup-module__inputs">
-				<AccountSelect />
+				<AccountSelect hasModuleAccess={ hasModuleAccess } />
 
-				<WebContainerSelect />
+				<WebContainerSelect hasModuleAccess={ hasModuleAccess } />
 
-				<AMPContainerSelect />
+				<AMPContainerSelect hasModuleAccess={ hasModuleAccess } />
 
 				<TagCheckProgress />
 			</div>
 
+			{ hasModuleAccess === false && (
+				<SettingsNotice
+					type={ TYPE_INFO }
+					Icon={ WarningIcon }
+					notice={ createInterpolateElement(
+						sprintf(
+							/* translators: 1: module owner's name, 2: module name */
+							__(
+								'%1$s configured %2$s and you don’t have access to this Tag Manager account. Contact them to share access or change the Tag Manager account.',
```

Comment:
Shouldn't "Tag Manager" here also use the translated module name instead of including it in the string?

Reply:
That's a good catch. Thanks, @tofumatt 👍

Reply:
It's so easy to miss these things 😅 

Thanks, looks good now 👍🏻